background_scripts/commands.coffee

```diff
@@ -74,6 +74,27 @@ Commands =
     Settings.set "passNextKeyKeys",
       (key for own key of @keyToCommandRegistry when @keyToCommandRegistry[key].command == "passNextKey" and 1 < key.length)
 
+  # Command options follow command mappings, and are of one of two forms:
+  #   key=value     - a value
+  #   key           - a flag
+  parseCommandOptions: (command, optionList) ->
+    options = {}
+    for option in optionList
+      parse = option.split "="
```

Comment:
We can use `option.split "=", 2` here to ignore any later `=` matches, and then the `else` case below becomes redundant.